### ISSUE TITLE
Runner#verifyRequest() produces wrong error code

### ISSUE THREAD

**Issue by nelsonxb** [NelsonCrosby/node-json-rpc#9]
When request is `null`, in the thrown error has `code === null`.

See test:

``` js
  request = null
  try { runner.verifyRequest(request) }
  catch (e) {
    t.is(e.code, -32600)
    t.true(e.message != null)
  }
```

```
  1. core › Runner: #verifyRequest() throws on not object

  t.is(e.code, -32600)
         |            
         null         

      Test.fn (core.js:233:7)
```

**Comment by nelsonxb** [comment]
Apparently, `typeof null === 'object'`. Because JavaScript.

**Comment by nelsonxb** [comment]
And the error that's actually being thrown is probably coming from `request.jsonrpc`, trying to read property of `null`, an `Error` which has no `code` property, hence the error message.

**Comment by nelsonxb** [comment]
Merged onto `v1.0-dev` in #12.
